`Missing plugin` failure while building indexer packages

## Description
While analyzing https://github.com/wazuh/wazuh-qa-automation/issues/5148, it has been detected that the indexer packages building procedure is failing due to:

```console
-> Failed installing file:/home/runner/work/wazuh-indexer/wazuh-indexer/artifacts/tmp/deb/../../plugins/wazuh-indexer-content-manager-5.0.0.latest.zip
-> Rolling back file:/home/runner/work/wazuh-indexer/wazuh-indexer/artifacts/tmp/deb/../../plugins/wazuh-indexer-content-manager-5.0.0.latest.zip
-> Rolled back file:/home/runner/work/wazuh-indexer/wazuh-indexer/artifacts/tmp/deb/../../plugins/wazuh-indexer-content-manager-5.0.0.latest.zip
Error: Exception in thread "main" java.lang.IllegalArgumentException: Missing plugin [wazuh-indexer-security-analytics], dependency of [wazuh-indexer-content-manager]
	at org.opensearch.plugins.PluginsService.addSortedBundle(PluginsService.java:594)
	at org.opensearch.plugins.PluginsService.sortBundles(PluginsService.java:557)
	at org.opensearch.plugins.PluginsService.checkJarHellForPlugin(PluginsService.java:401)
	at org.opensearch.tools.cli.plugin.InstallPluginCommand.jarHellCheck(InstallPluginCommand.java:834)
	at org.opensearch.tools.cli.plugin.InstallPluginCommand.loadPluginInfo(InstallPluginCommand.java:811)
	at org.opensearch.tools.cli.plugin.InstallPluginCommand.installPlugin(InstallPluginCommand.java:846)
	at org.opensearch.tools.cli.plugin.InstallPluginCommand.execute(InstallPluginCommand.java:277)
	at org.opensearch.tools.cli.plugin.InstallPluginCommand.execute(InstallPluginCommand.java:251)
	at org.opensearch.common.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:110)
	at org.opensearch.cli.Command.mainWithoutErrorHandling(Command.java:138)
	at org.opensearch.cli.MultiCommand.execute(MultiCommand.java:104)
	at org.opensearch.cli.Command.mainWithoutErrorHandling(Command.java:138)
	at org.opensearch.cli.Command.main(Command.java:101)
	at org.opensearch.tools.cli.plugin.PluginCli.main(PluginCli.java:66)
Error: Process completed with exit code 1.
```

## Evidence

- indexer-rpm-x86_64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/20183430546
- indexer-deb-amd64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/20183430478
- indexer-deb-arm64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/20183430487
- indexer-rpm-aarch64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/20183430599

--- a/build-scripts/assemble.sh
+++ b/build-scripts/assemble.sh
@@ -39,9 +39,9 @@ else
     )
     wazuh_plugins=(
         "wazuh-indexer-setup"
+        "wazuh-indexer-security-analytics" # Flagged as Trojan by some antivirus software
         "wazuh-indexer-content-manager"
         "wazuh-indexer-reports-scheduler"
-        "wazuh-indexer-security-analytics" # Flagged as Trojan by some antivirus software
     )
 fi
 

